src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -16,6 +16,7 @@ namespace System.Diagnostics
     ///     used in conjunction with the <see cref='System.Diagnostics.Process'/>
     ///     component.
     /// </devdoc>
+    [DebuggerDisplay("FileName={FileName}, Arguments={BuildArguments()}, WorkingDirectory={WorkingDirectory}")]
```

Comment:
Would it be better to quote them? Or something to indicate the boundaries.
```suggestion
    [DebuggerDisplay(@"FileName=""{FileName}"", Arguments=""{BuildArguments()}"", WorkingDirectory=""{WorkingDirectory}""")]
```

Reply:
@danmosemsft it looks like the `[DebuggerDisplay]` (or VS) quotes the properties by default. With your suggestion I had two quotes. I've reverted it.

![obraz](https://user-images.githubusercontent.com/6011991/85544920-bda06880-b61b-11ea-954c-5121a0d8161d.png)

![obraz](https://user-images.githubusercontent.com/6011991/85544936-c133ef80-b61b-11ea-83f3-318458c9ef71.png)

Was putting following thing to clipboard:

```log
FileName=""cmd.exe"", Arguments=""/c start /NODE 0 /AFFINITY 08 C:\\Windows\\System32\\notepad.exe"", WorkingDirectory=""""
```

Now it's:

```log
FileName="cmd.exe", Arguments="/c start /NODE 0 /AFFINITY 08 C:\\Windows\\System32\\notepad.exe", WorkingDirectory=""
```